src/com/google/javascript/jscomp/CommandLineRunner.java

```diff
@@ -682,7 +682,12 @@
     @Option(name = "--print_source_after_each_pass",
         hidden = true,
         usage = "Whether to iteratively print resulting JS source per pass.")
-        private boolean printSourceAfterEachPass = false;
+    private boolean printSourceAfterEachPass = false;
+
+    @Option(name = "--module_resolution",
+        hidden = false,
+        usage = "Specifies how the compiler locates modules. Defaults to NODE.")
```

Comment:
The current compiler behavior isn't standard in any way. It doesn't match any other module loading system. The expectation of users (especially new users) is that it works like Node because to date the biggest use case is bundling.